packages/ketcher-react/src/script/editor/operations/simpleObject.ts

```diff
@@ -0,0 +1,337 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import Vec2 from '../../util/vec2'
+import Base, { invalidateItem, OperationType } from './base'
+import { ReSimpleObject } from '../../render/restruct'
+import { SimpleObject, SimpleObjectMode } from 'src/script/chem/struct'
+import scale from "../../util/scale";
+import util from "../../render/util";
+
+const tfx = util.tfx
```

Comment:
what is 'tfx' ? what does it mean?

Reply:
It alias for Float.toFixed, to compare more precise